text/0155-external-modules.md

```diff
@@ -0,0 +1,106 @@
+# Instrumentation Ecosystem Management
+
+Proposal how to leverage wider community contributing instrumentations and other packages.
+
+## Motivation
+
+For OpenTelemetry to become a de-facto standard in observability there must exist a vast ecosystem of OpenTelemetry components.
+Including integrations with various libraries and frameworks in all languages supported by OpenTelemetry.
+We cannot possibly expect that all these integrations will be provided by the core maintainers of OpenTelemetry.
+We have to have a way to leverage a wider community while still providing our end-users with some way to discover
+all available Otel components together with some visibility into their quality.
+
+## Explanation
+
+[OpenTelemetry Registry](https://opentelemetry.io/registry/) serves as a central catalogue of all known Otel components,
+both provided by core maintainers of the project and any third party.
+
+In order for a component to be included into Registry its authors have to fill a [self-assessment form](#registry-self-assessment-form).
+
+Registry should allow a clear visibility if component came from a repository in OpenTelemetry organisation.
```

Comment:
I think having strong registry is critical to this OTEP, so it makes sense to pay more attention to it and list the requirements it must satisfy, i.e. what required information it must capture about each instrumentation component, e.g. things like
  * location url
  * language
  * target instrumented library
  * some compatibility info, like the versions of OTEL & the instrumented library
  * license
  * dimensions of the self-assessment form
  * official status (maintained by SIG | in contrib | 3rd party)
  * any quality indicators we could standardize on (e.g. test coverage)
  * "popularity" (e.g. github stars)
  * dev stats (date of creation, date of last release, date of last commit)

The website can have a daily cron job to rebuild the registry with fresh info

Reply:
Do you think this list of all information pieces should be agreed upon and finalized in this OTEP? Or this will be a part of the effort of implementing this OTEP in the Registry? I think the latter.

Reply:
I think it should be part of this OTEP, since it's a list of requirements to the registry that *this* OTEP depends on (doesn't have to be all of the above, but there's a minimal required set).

Reply:
Added.